src/main/java/seedu/address/model/person/Customer.java

```diff
@@ -0,0 +1,100 @@
+package seedu.address.model.person;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Objects;
+import java.util.Set;
+
+import seedu.address.model.tag.Tag;
+import seedu.address.model.tag.UniqueTagList;
+
+/**
+ * Represents a Person in the address book.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+public class Customer extends Person {
+
+    private final Name name;
+    private final Phone phone;
+    private final Email email;
+    private final Address address;
+
+    private final UniqueTagList tags;
+
+    private final String moneyOwed;
+    private final String interestRate;
+
+    /**
+     * Every field must be present and not null.
+     */
+    public Customer(Name name, Phone phone, Email email, Address address, Set<Tag> tags, String moneyOwed,
+                    String interestRate) {
+        requireAllNonNull(name, phone, email, address, tags, moneyOwed, interestRate);
+        this.name = name;
+        this.phone = phone;
+        this.email = email;
+        this.address = address;
+        this.tags = new UniqueTagList(tags);
+        this.moneyOwed = moneyOwed;
+        this.interestRate = interestRate;
+    }
```

Comment:
Could we can call the `Person` constructor to do the assignment of attributes inherited from the `Person` parent class?

Reply:
we could but I looked through the rest of the code and it might make more sense to/be easier to construct a Customer directly rather than construct a Person than a Customer. I'm not sure though, need to discuss this further.